packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -411,14 +411,15 @@ const Services = WebexPlugin.extend({
    * @param {object} query
    * @param {string} query.email - A standard format email.
    * @param {string} query.orgId - The user's OrgId.
+   * @param {boolean} forceRefresh - Boolean to bypass u2c cache control header
```

Comment:
```suggestion
   * @param {boolean} [forceRefresh=false] - Boolean to bypass u2c cache control header
```

Reply:
Added tests. Let me know if it's not sufficient. Was attempting to tests the request itself to see if timestamp is apart of the query parameter as part of Andrew's PR, but that's tightly coupled with webex request.